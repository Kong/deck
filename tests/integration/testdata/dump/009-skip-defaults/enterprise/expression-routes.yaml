_format_version: "3.0"
services:
- connect_timeout: 60000
  enabled: true
  host: example.com
  name: service-1
  port: 80
  protocol: http
  read_timeout: 60000
  retries: 5
  routes:
  - expression: ' net.src.ip == 192.168.1.0 && net.protocol == "http" && http.method
      == "GET"'
    https_redirect_status_code: 426
    name: r2
    preserve_host: false
    priority: 0
    protocols:
    - http
    - https
    request_buffering: true
    response_buffering: true
    strip_path: true
    tags:
    - tag1
    - tag2
  tags:
  - tag1
  - tag2
  write_timeout: 60000
