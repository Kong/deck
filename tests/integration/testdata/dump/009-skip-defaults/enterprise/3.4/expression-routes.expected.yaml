_format_version: "3.0"
services:
- host: example.com
  name: service-1
  routes:
  - expression: ' net.src.ip == 192.168.1.0 && net.protocol == "http" && http.method
      == "GET"'
    name: r2
    tags:
    - tag1
    - tag2
  tags:
  - tag1
  - tag2
